.gitignore

```diff
@@ -21,9 +21,10 @@ cmake/CMakeFiles/
 *.png
 *.pyc
 
+include/kiss/
```

Comment:
Please remove include/kiss/ and include/healpix_base/ from the .gitignore file.

Those files are likely tracked because you had the install_prefix set to the CRPropa source path. If you change that to, e.g., usr/local or something not within the git repository there is no need to git-ignore these folders.